hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -77,13 +77,39 @@ Or put command etc. In other words, all programs like Hive, Spark, and Distcp wi
 Please note that any keys created/deleted in the bucket using methods apart from OzoneFileSystem will show up as directories and files in the Ozone File System.
 
 Note: Bucket and volume names are not allowed to have a period in them.
-Moreover, the filesystem URI can take a fully qualified form with the OM host and port as a part of the path following the volume name.
-For example,
+Moreover, the filesystem URI can take a fully qualified form with the OM host and an optional port as a part of the path following the volume name.
+For example, you can specify both host and port:
 
 {{< highlight bash>}}
 hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:5678/key
 {{< /highlight >}}
 
+When the port number is not specified, it will be retrieved from config key `ozone.om.address`.
+For example, we have `ozone.om.address` configured as following in `ozone-site.xml`:
+
+{{< highlight xml >}}
+  <property>
+    <name>ozone.om.address</name>
+    <value>0.0.0.0:6789</value>
+  </property>
+{{< /highlight >}}
+
+When we run command:
+
+{{< highlight bash>}}
+hdfs dfs -ls o3fs://bucket.volume.om-host.example.com/key
+{{< /highlight >}}
+
+The above command is essentially equivalent to:
+
+{{< highlight bash>}}
+hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
+{{< /highlight >}}
+
+Note only the port number in the config is relevant. The host name in config `ozone.om.address` is ignored in this case.
```

Comment:
Can we reword this as:
Note: In the above case, port number from the config is used, whereas hostname in the config `ozone.om.address` is ignored.

Reply:
done